spyder/plugins/application/widgets/status.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Status widget for Spyder updates.
+"""
+
+# Standard library imports
+import logging
+
+# Third party imports
+from qtpy.QtCore import Slot
+
+# Local imports
+from spyder import get_versions
+from spyder.api.widgets.status import StatusBarWidget
+from spyder.config.base import _, is_pynsist
+from spyder.plugins.application.widgets.install import (
+    UpdateInstallerDialog, NO_STATUS, DOWNLOADING_INSTALLER, INSTALLING,
+    FINISHED, PENDING, CHECKING, CANCELLED)
+from spyder.utils.icon_manager import ima
```

Comment:
New imports for the menu to show the `Check for updates...` action:
```suggestion
from spyder.utils.icon_manager import ima
from spyder.utils.qthelpers import add_actions, create_action
```